Board tweaks

ArkadiuszRaj opened the pull request:
- dedicated input of 5V  to allow control of external 24V supply
- minor tracks adjustment / cleaning
- added CAN terminal (PD0-PD1)
- SD card switched to SPI mode and moved to back side (below DC buck converter) connected to SPI3 interface
- removed EXT4 header


![obraz](https://user-images.githubusercontent.com/16072469/104806457-ff6f4700-57d7-11eb-8df5-2a9815620bf9.png)
![obraz](https://user-images.githubusercontent.com/16072469/104806470-131aad80-57d8-11eb-8976-3e71657d0edd.png)

Lets treat it as intermediate work, as I believe some additional tracks tweaking will be needed, especially to check power distribution.
 
TO DO:
- TFT screen interface (additiinal usart)
- WI-FI interface (simple header for esp8266-01s)
- most probably dedicated STM32F4 IAP bootloader must be created to support firmware update via SD card / FAT / SPI3

ghent360 commented:
Not a fan of the micro sd on the bottom because it may be PITA to solder w/o glue on the top layer components, but it is ok for now. I'll see what I can move around to put it on that side of the board.

ghent360 commented:
Maybe if I move the fan voltage selector up and put the dc converter on the bottom - it is eazy to hand solder vs the micro-sd that needs hot air. I'll give it some more though.

Thank you for the contribution.

ArkadiuszRaj commented:
Good idea with moving buck pcb to bottom.
Some more vias would be needed to secure current flow between layers.
I will modify the PCB shortly.

BTW. I have added 5th pin to Serial header and renamed it to TFT. There is
also one free gpio line I plan to use as dedicated mosfet driven ATX power
supply.

Also I will prepare add on board to use ESP-12F as Wi-Fi connection.

W dniu sob., 16.01.2021 o 14:26 ghent360 <notifications@github.com>
napisał(a):

> Maybe if I move the fan voltage selector up and put the dc converter on
> the bottom - it is eazy to hand solder vs the micro-sd that needs hot air.
> I'll give it some more though.
>
> Thank you for the contribution.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ghent360/PrntrBoardV2/pull/4#issuecomment-761563039>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD2T6FMT2G2ZEPPHIVQWEP3S2GHYJANCNFSM4WEAIKTA>
> .
>
